documentation/architecture/decisions/0001-environments.md

```diff
@@ -14,14 +14,19 @@ production work and our users.
 
 ## Decision
 
-We have decided to have two separate environments, staging and production. The staging
-environment will be where the tools team and other people working on monitoring
-and metrics itself will test their changes. The production environment will run
-all of our users monitoring and metrics and poll each of their environments.
+We have decided to have N+2 separate environments: development,
+staging and production. In development, we can create as many separate
+stacks as we want. The staging environment will be where the tools team
+and other people working on monitoring and metrics itself will test
```

Comment:
`other people working on monitoring and metrics itself`. I'm not quite sure who this is referring to? Other RE devs who may not be in the tools team who are responsible for upkeep of prometheus? Or are you referring to our end users such as DGU who will use our staging environment to test their changes?

Based on `Users will test their prometheus related changes in our production environment and will not have access  to the staging environment.` in the consequences section I'm guessing it's not the later. Therefore if I've understood correctly, it feels like it might be sufficient to just say `The staging environment will be where the tools team will test their changes`.

Reply:
i agree and will make the suggested change